src/ast.hpp

```diff
@@ -103,6 +103,14 @@ struct FunctionLiteral : public AST {
 	    : AST {ast_type::FunctionLiteral} {}
 };
 
+struct ShortFunctionLiteral : public AST {
+	std::unique_ptr<AST> m_body;
+	std::vector<std::unique_ptr<AST>> m_args;
+
+	ShortFunctionLiteral()
+	    : AST {ast_type::ShortFunctionLiteral} {}
+};
+
```

Comment:
What is the benefit of this?

Reply:
Less ad-hoc code in the parser, it would be helpful in writing tools, like a code formatter.

Reply:
Fair enough